data-prepper-main/src/main/java/org/opensearch/dataprepper/DataPrepperExecute.java

```diff
@@ -19,7 +19,14 @@ public class DataPrepperExecute {
     public static void main(final String ... args) {
         java.security.Security.setProperty("networkaddress.cache.ttl", "60");
 
-        final ContextManager contextManager = new ContextManager(args);
+        final String dataPrepperHome = System.getProperty("data-prepper.dir");
+        if (dataPrepperHome == null) {
+            throw new RuntimeException("Data Prepper home directory (data-prepper.dir) not set in system properties.");
+        }
+
+        final String dataPrepperPipelines = dataPrepperHome + "/pipelines/pipelines.yaml";
```

Comment:
We should probably use `Paths` here instead of adding strings.

https://docs.oracle.com/javase/8/docs/api/java/nio/file/Path.html#resolve-java.lang.String-

Reply:
Also fixed in [46460e9](https://github.com/opensearch-project/data-prepper/pull/1737/commits/46460e974fdcf536de2fb91782fb59463ff32169)